src/collective/isotope/browser/templates/isotope.pt

```diff
@@ -92,7 +92,7 @@ $(window).load(function(evt) {
                                          item_wf_state item/review_state;
                                          item_wf_state_class python:'state-' + view.normalizeString(item_wf_state);
                                          item_link python:item_type in view.use_view_action and item_url+'/view' or item_url;
-                                         item_has_image python:view.has_image(item);
+                                         item_has_image python:item.getIcon;
```

Comment:
Just wanting to be sure, @fulv, before I merge.  This looks like simply the presence of a `getIcon` attribute on any item is enough to prove that the item does in fact have an image field.  Is that true?  If so, I'm happy to merge this, and thanks for the fix!